common/openstack/osp-keystone-api-v3.adoc

```diff
@@ -1 +1,3 @@
-With Keystone API v3, domains are used to determine administrative boundaries of service entities in OpenStack. Domains allow you to group users together for various purposes, such as setting domain-specific configuration or security options. For more information, see https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/architecture-guide/chapter-1-components#comp-identity[OpenStack Identity (keystone)] in the Red Hat OpenStack Platform _Architecture Guide_.
+* With Keystone API v3, domains are used to determine administrative boundaries of service entities in OpenStack. Domains allow you to group users together for various purposes, such as setting domain-specific configuration or security options. For more information, see https://access.redhat.com/documentation/en/red-hat-openstack-platform/8/architecture-guide/chapter-1-components#comp-identity[OpenStack Identity (keystone)] in the Red Hat OpenStack Platform _Architecture Guide_.
+
+* The provider you are creating will be able to see inventory from the given domain only. To see inventory for other domains, add it as another cloud provider. For more information on domain management in OpenStack, see https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/8/html-single/users_and_identity_management_guide/#Domains[Domain Management] in the Red Hat OpenStack Platform _Users and Identity Management Guide_.
```

Comment:
Please replace ```inventory``` keyword with ```projects```. It's less generic, but propably more clear to users, thanks.

Otherwise looks good to me!